api/memory/graphiti_tool.py

```
@@ -243,48 +170,110 @@
                 reference_time=datetime.now(),
-                source_description=f"Personal memory for user {user_id}"
+                source_description=f"Graph-oriented facts about Database: {database_name} from User: {user_id} interaction"
             )
-                
+            
+            update_user_task = self.update_user_information(conversation, history=history)
+            
+            # Wait for both operations to complete
+            await asyncio.gather(add_episode_task, update_user_task)
+
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)